components/style/values/computed/image.rs

```diff
@@ -454,3 +455,73 @@ impl ToComputedValue for specified::LengthOrPercentageOrKeyword {
         }
     }
 }
+
+#[derive(Clone, Debug, PartialEq)]
+#[cfg_attr(feature = "servo", derive(HeapSizeOf))]
+pub enum AngleOrCorner {
```

Comment:
No, it was imported as `computed::AngleOrCorner` and so I suggested moving it to `computed/mod.rs` to fix the build failure. We have plenty of other options - I'm cool with anything :)